Library/Location.swift

```diff
@@ -60,25 +59,25 @@ struct Location {
     }
 
     var isNight: Bool {
-        return self.sunPhase == .night
+        return self.sunPhase == .nightEnd
```

Comment:
`.night` starts where a day starts, night end is basically the second time there's night that day.